tests/ast/data/df_first.test

```diff
@@ -0,0 +1,19 @@
+## TEST CASE
+
+df = session.table("test_table")
+
+df = df.first(-5)
+
+df = df.first(2)
+
+df = df.first()
+
+## EXPECTED OUTPUT
+
+res1 = session.table("test_table")
+
+res2 = res1.first(-5)
```

Comment:
Does this test actually pass? I don't see `first` in the unparser (current branch + your other diff).

Reply:
No, I can't run the tests - I spoke about this with @sfc-gh-lspiegelberg  yesterday and I believe that the tests don't run on the `server-side-snowpark` branch either. Leonhard mentioned it's probably because the jar file is not a fat jar.